website/docs/reference/commands/version.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "About dbt --version"
```

Comment:
We should link to this [here](https://docs.getdbt.com/reference/global-cli-flags#version) or maybe even remove that section from that page